### PR TITLE
Convert issue templates to issue forms

### PR DESCRIPTION
**Pull request by mcdonnnj**
This updates the existing GitHub issue templates to use the newer issue forms capability.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report_form.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+      placeholder: A list of steps to reproduce the described bug.
+    validations:
+      required: true
+  - type: dropdown
+    id: browser
+    attributes:
+      label: Browser
+      description: What browser are you using?
+      multiple: true
+      options:
```
Comment (henrik9999) [inline .github/ISSUE_TEMPLATE/bug_report_form.yml:25]: Thanks for your PR, but im not really sure about this one I dont think all people know that when they use opera or something that its a chrome based browser so maybe this should be a text input or if possible a dropdown where you can additionaly write something else if its not one of the two.

**Review by henrik9999 — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug_report_form.yml (hunk 1 above)
Comment (mcdonnnj) [reply .github/ISSUE_TEMPLATE/bug_report_form.yml:25]: Sure I can change it to a regular text input. I was just mirroring the style of the existing templates for these when I converted them.

Comment (henrik9999) [reply .github/ISSUE_TEMPLATE/bug_report_form.yml:25]: I think it was more meant of an example on the old version

Comment (henrik9999) [reply .github/ISSUE_TEMPLATE/bug_report_form.yml:25]: @mcdonnnj are you planning to do the change with the text input?

Comment (mcdonnnj) [reply .github/ISSUE_TEMPLATE/bug_report_form.yml:25]: @henrik9999 Ah, sorry this completely fell off my radar. Yes, I will update my branch tonight or tomorrow at the latest. Thanks for the bump!

Comment (mcdonnnj) [reply .github/ISSUE_TEMPLATE/bug_report_form.yml]: @henrik9999 Please see [06932e2](https://github.com/MALSync/MALSync/pull/1079/commits/06932e2add919ccaed5c15e5e68f34e77bfbf61c) and let me know if that satisfies your concerns.

**Review by henrik9999 — APPROVED** [review]

**Pull request merged**